inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -330,21 +332,19 @@ def filter_keywords(obj, eng):
 @with_debug_logging
 def prepare_keywords(obj, eng):
     """Prepares the keywords in the correct format to be sent"""
-    prediction = obj.extra_data.get('keywords_prediction', {})
-    if not prediction:
-        return
-
     keywords = obj.data.get('keywords', [])
-    for keyword in prediction.get('keywords', []):
-        # TODO: differentiate between curated and guessed keywords
-        keywords.append(
-            {
-                'value': keyword['label'],
-                'source': 'curator' if keyword.get('curated') else 'magpie',
-            }
-        )
-
-    obj.data['keywords'] = keywords
+    if obj.data.get('core'):
+        extracted_keywords = obj.extra_data.get('extracted_keywords', [])
+        for keyword in extracted_keywords:
+            if keyword:
+                keywords.append(
+                    {
+                        "value": keyword,
+                        "schema": "INSPIRE"
```

Comment:
sorry, small addition: it would be nice to add `"source": "classifier"` here so we know where they come from

Reply:
:+1: